etc/linters.json

```diff
@@ -0,0 +1,28 @@
+[
```

Comment:
We should order this so the faster ones are first - it will give us a (very minor) performance improvement.  I think:
- ruff
- dockerfile_lint
- markdown-link-check
- cspell

Reply:
Done